Makefile

```diff
@@ -58,3 +80,21 @@ test:
 .PHONY: format
 format:
 	@elm-format --yes src/ tests/
+
+.PHONY: bump-deps
+bump-deps: clean
```

Comment:
Thoughts on `upgrade` instead of `bump-deps`?

It's a little shorter and easier to remember, but it could have an unintended side effect of someone misinterpreting what the purpose for the command is 🤷‍♂️ 

I bring it up because I'm currently working on something similar and want to remain consistent 👍

Reply:
yea, that's what i had before and then i switched for brevity. i'm fine either way though.

Reply:
Hmmm... maybe for now we stick with `bump-deps` and consider `upgrade` later.

I was using `update` and `upgrade` terminology from an adoption of Homebrew.

But that might not be the best decision due to those who may be unfamiliar 🤷‍♂️

Reply:
only reason i'm hesitant is because there might be a use for a target like that that does more than upgrade dependencies. but it could still work.

Reply:
Sure 👍  

Unless others feel strongly, I'm good to use `bump` over `upgrade`.

Tell you what, I'll leave it up to you and then follow suit with whatever you decide 😉

Reply:
I think I’ll leave it as is and reserve just bump for potential future usage.